lib/registries/addon/styles/variables.scss

```diff
@@ -1,5 +1,6 @@
 @import 'app/styles/variables';
 
+$selected-blue: rgba(51, 122, 183, 0.2); // Used to show current page
```

Comment:
This is `$primary-blue` with the opacity dialed down -- would be nicer to express that relation:
```scss
$primary-blue: #337ab7;
$selected-blue: rgba($primary-blue, 0.2);
```